test.py

```diff
@@ -946,6 +967,51 @@ async def run(self, options: argparse.Namespace) -> Test:
         return self
 
 
+class ToolTest(Test):
+    """Run a pytest collection of cases against a standalone Scylla"""
```

Comment:
The phrase "standalone Scylla" is misleading because you said above these tests don't need Scylla. Maybe you meant to say "mock Scylla"?

Reply:
Rephrased the comment in v4.